pkg/lib/pb/model/protos/models.proto

```diff
@@ -50,6 +50,7 @@ enum SmartBlockType {
     FileObject = 0x215;
 
     NotificationObject = 0x217;
+    DeviceObject = 0x218;
```

Comment:
I think Page type is sufficient for this task, as you probably don't need special behavior for smartblock

Reply:
Fixed